docs/user-guide/pod-security-policy/index.md

```diff
@@ -163,5 +163,6 @@ following
 
 ## Working With RBAC
 
-Use PodSecurityPolicy to control access to privileged containers based on role and groups.
-(see [more details](https://github.com/kubernetes/kubernetes/blob/master/examples/podsecuritypolicy/rbac/README.md)).
+Kubernetes supports role based PodSecurityPolicy to control various privileges on deployed containers. To enable the use of RBAC with PodSecurityPolicy for objects e.g. Deployment, ReplicaSet, it is required to run [Controller Manager](https://kubernetes.io/docs/admin/kube-controller-manager/) against APIServer's secured port and have `--use-service-account-credentials` option added to its command line arguments. PodSecurityPolicy uses the union of all accessible policies and apply the highest privileges avaiblable during the pod deployment. In general, when you run a Deployment object, it is Controller Manager who deploys the pod, so if it is running against unsecured port, it will have access to all the policies and you will not be able to effectively apply RBAC model.
```

Comment:
`
Use PodSecurityPolicy to control access to privileged containers. In Kubernetes 1.5+, access to different PodSecurityPolicy objects can be controlled via authorization. To limit access to PodSecurityPolicy objects for pods created via a Deployment, ReplicaSet, etc, the [Controller Manager](/docs/admin/kube-controller-manager/) must be run against the secured API port, and must not have superuser permissions. 

PodSecurityPolicy authorization uses the union of all policies available to the user creating the pod and the service account specified on the pod. When pods are created via a Deployment, ReplicaSet, etc, it is Controller Manager that creates the pod, so if it is running against the unsecured API port, all PodSecurityPolicy objects would be allowed, and you could not effectively subdivide access.
`